string data
string action
string changes
evd_ros_core/Version currentTag
evd_ros_core/Version previousTag
